src/cloud-api-adaptor/podvm-mkosi/Makefile

```diff
@@ -64,8 +54,9 @@ endif
 		--build-arg VERIFY_PROVENANCE=$(VERIFY_PROVENANCE) \
 		$(if $(AUTHFILE),--build-arg AUTHFILE=$(AUTHFILE),) \
 		$(if $(DEFAULT_AGENT_POLICY_FILE),--build-arg DEFAULT_AGENT_POLICY_FILE=$(DEFAULT_AGENT_POLICY_FILE),) \
-		-o type=local,dest="./resources/binaries-tree" \
-		-f ../podvm/Dockerfile.podvm_binaries.fedora ../../
+		$(if $(filter $(PUSH),true),,-o type=local,dest="./resources/binaries-tree") \
+		$(DOCKER_OPTS) \
+		-f ../podvm/Dockerfile.podvm.fedora ../../
```

Comment:
Hi @stevenhorsman !

Shouldn't it still use `Dockerfile.podvm_binaries.fedora`?

Reply:
Sorry, I renamed this last minute (I created a new one in order to keep the old code untouched before deciding this didn't make sense). I will fix and re-test. thanks for the spot!